apps/web/src/pages/DirectiveForms/DigitalServicesContractingQuestionnaire/sections/RequirementsSection.tsx

```diff
@@ -0,0 +1,438 @@
+    isHasPersonnelRequirementsNo,
+    hasOtherRequirementsIsYes,
+  ]);
+
+  return (
+    <TableOfContents.Section
+      id={PAGE_SECTION_ID.CONTRACT_REQUIREMENTS}
+      data-h2-padding-top="base(x2)"
+    >
+      <Heading data-h2-margin="base(0, 0, x1, 0)" level="h3">
+        {intl.formatMessage(
+          getSectionTitle(PAGE_SECTION_ID.CONTRACT_REQUIREMENTS),
+        )}
+      </Heading>
+      <div
+        data-h2-display="base(flex)"
+        data-h2-flex-direction="base(column)"
+        data-h2-gap="base(x1)"
+      >
+        <CompoundQuestion
+          title={intl.formatMessage({
+            defaultMessage: "Description of the work required",
+            id: "T0JO/F",
+            description:
+              "title for _work requirement description_ textbox in the _digital services contracting questionnaire_",
+          })}
+          introduction={
+            <>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "What are the tasks that the contractor is expected to perform within the contract?",
+                  id: "//mFwp",
+                  description:
+                    "Context for _work requirement description_ textbox in the _digital services contracting questionnaire_",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage: "For example, the contractor will:",
+                  id: "I+GYIo",
+                  description:
+                    "Context for _work requirement description_ textbox in the _digital services contracting questionnaire_",
+                })}
+              </p>
+              <ul>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "tag and augment provided datasets as may be required for reporting",
+                    id: "eeyR6R",
+                    description:
+                      "Context example for _work requirement description_ textbox in the _digital services contracting questionnaire_",
+                  })}
+                </li>
+                <li>
+                  {intl.formatMessage({
+                    defaultMessage:
+                      "manipulate and perform calculations against the dataset",
+                    id: "tDwAB0",
+                    description:
+                      "Context example for _work requirement description_ textbox in the _digital services contracting questionnaire_",
+                  })}
+                </li>
+              </ul>
+            </>
+          }
+          inputElement={
+            <TextArea
+              id="workRequirementDescription"
+              name="workRequirementDescription"
+              wordLimit={200}
+              label={labels.workRequirementDescription}
+              rules={{ required: intl.formatMessage(errorMessages.required) }}
+            />
```

Comment:
I think we've got a pattern building of the word limit being contingent on locale, more words for the French. Not sure if needed for this form

Reply:
Hmm, good thought.  For now we're matching the original form which has the same limit in English and French but I wouldn't be surprised if this changed at some point.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/8c61641b-69c8-4324-a36c-df475cb57ff9)

Reply:
Response from Rosita: keep it as is for now